src/core/manager.h

```diff
@@ -36,9 +36,9 @@ class Manager {
   Manager();
   ~Manager();
 
-  DownloadList*       download_list()                     { return m_download_list.get(); }
-  DownloadStore*      download_store()                    { return m_download_store.get(); }
-  FileStatusCache*    file_status_cache()                 { return m_file_status_cache.get(); }
+  const auto&         download_list() const               { return m_download_list; }
+  const auto&         download_store() const              { return m_download_store; }
+  const auto&         file_status_cache() const           { return m_file_status_cache; }
```

Comment:
This changes the behavior of the code, not sure I want this.

I'd rather use the old pointers in these cases.

Reply:
Does it really matter? It’s still a non const pointer.